src/app/workflow/workflow.component.html

```diff
@@ -13,241 +13,268 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-
-<div class="row m-1" *ngIf="error  || missingWarning">
-  <div class="col-md-12" *ngIf="!isWorkflowPublic">
-    <mat-card class="alert alert-warning" ng-class="!editMode ? 'push-top' : ''" role="alert" *ngIf="missingWarning">
-      <button type="button" class="close" data-dismiss="alert" ng-click="missingWarning = false">
-        &times;
-      </button>
-      <mat-icon>warning</mat-icon>
-    </mat-card>
+<div *ngIf="workflow">
+  <div class="row m-1" *ngIf="error  || missingWarning">
+    <div class="col-md-12" *ngIf="!isWorkflowPublic">
+      <mat-card class="alert alert-warning" ng-class="!editMode ? 'push-top' : ''" role="alert" *ngIf="missingWarning">
+        <button type="button" class="close" data-dismiss="alert" ng-click="missingWarning = false">
+          &times;
+        </button>
+        <mat-icon>warning</mat-icon>
+      </mat-card>
+    </div>
   </div>
-</div>
-<div class="row m-1" *ngIf="showRedirect">
-  <div class="col-md-10 col-md-offset-1">
-    <h2 class="text-center">The workflow linked to could not be found!</h2>
-    <p class="text-center">
-      Workflows have been updated to include the Git registry in their path for identification purposes. The workflow linked to did not have
-      a valid Git registry in the path. In most cases you were looking for <strong><a href="/workflows/{{githubPath}}">{{githubPath}}</a></strong>. If not then it
-      may be one of the following.
-    </p>
-    <ul class="text-center list-inline">
-      <li><a href="/workflows/{{bitbucketPath}}">{{bitbucketPath}}</a></li>
-      <li><a href="/workflows/{{gitlabPath}}">{{gitlabPath}}</a></li>
-    </ul>
+  <div class="row m-1" *ngIf="showRedirect">
+    <div class="col-md-10 col-md-offset-1">
+      <h2 class="text-center">The workflow linked to could not be found!</h2>
+      <p class="text-center">
+        Workflows have been updated to include the Git registry in their path for identification purposes. The workflow
```

Comment:
I know it was like this already, but should this be `GitHub registry` instead of `Git registry`?